templates/fields.html.twig

```diff
@@ -37,7 +37,7 @@
 
 {% if not already_wrapped and not dropdown_item%}
 
-{% set class = item.isNewItem() ? 'col-xxl-12' : 'col-xxl-9' %}
+{% set class = item.isNewItem() or item is instanceof('User') ? 'col-xxl-12' : 'col-xxl-9' %}
```

Comment:
For existing users, it is well aligned, but for new users, there is a slight misalignment:

<img width="947" height="339" alt="Image" src="https://github.com/user-attachments/assets/bf13eaae-8151-4a0e-b8c0-062f18dd77d0" />